common/requirements.txt

```diff
@@ -0,0 +1,21 @@
+#
+# This file is autogenerated by pip-compile
+# To update, run:
+#
+#    pip-compile --output-file requirements.txt requirements.in
+#
+boto3==1.4.4
```

Comment:
So re: our conversation in the previous PR, we are using boto3 instead of boto 2 because this does not deal with Celery workers?